android/build.gradle

```diff
@@ -23,12 +23,14 @@ android {
 }
 
 dependencies {
+  def googlePlayServicesVersion = project.hasProperty('googlePlayServicesVersion') ? project.googlePlayServicesVersion : DEFAULT_GOOGLE_PLAY_SERVICES_VERSION
```

Comment:
What's the difference between `project` and `rootProject` described here — https://github.com/invertase/react-native-firebase/blob/master/android/build.gradle#L84?
Should not `DEFAULT_GOOGLE_PLAY_SERVICES_VERSION` be defined?

Reply:
I have added DEFAULT_GOOGLE_PLAY_SERVICES_VERSION.
Not quite sure about the difference between $project and $rootProject. By looking at other projects it appears that `$rootProject` is used in the `app/build.gradle` whereas the `$project` keyword is used for libraries and `android/build.gradle`